src/js/components/BreadcrumbSegment.js

```diff
@@ -0,0 +1,58 @@
+import mixin from 'reactjs-mixin';
+/* eslint-disable no-unused-vars */
+import React, {PropTypes} from 'react';
+/* eslint-enable no-unused-vars */
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import BreadcrumbLink from './BreadcrumbLink';
+
+class BreadcrumbSegment extends mixin(StoreMixin) {
```

Comment:
Could you please explain why this breadcrumb segment component needs to extend the StoreMixin? I would prefer keeping this component store-agnostic and solving the problem using props if possible.